ui/pages/settings/index.scss

```diff
@@ -26,7 +26,6 @@
         text-overflow: ellipsis;
         overflow: hidden;
         white-space: nowrap;
-        max-width: 250px;
```

Comment:
@adonesky1 was there a case this was needed for in particular?

Reply:
Ah, now I see it served a purpose for very long contact names, so this needs to not be removed then but rather re-worked for normal use cases

Reply:
yes I didn't do this elegantly before. But now removing it is causing thi
<img width="358" alt="Screen Shot 2021-09-24 at 9 58 30 AM" src="https://user-images.githubusercontent.com/34557516/134696588-73d323d5-fbc0-4cd5-87e7-7b9d73add17f.png">
<img width="358" alt="Screen Shot 2021-09-24 at 9 58 24 AM" src="https://user-images.githubusercontent.com/34557516/134696592-15523043-da00-47f4-ada1-891c63413e80.png">
s:

Reply:
@adonesky1 I pushed changes that should cover all cases:
<img width="358" alt="Screen Shot 2021-09-24 at 8 23 28 AM" src="https://user-images.githubusercontent.com/8732757/134700458-b2f037d2-801d-407e-ad92-49eef70cacf4.png">
<img width="575" alt="Screen Shot 2021-09-24 at 8 23 03 AM" src="https://user-images.githubusercontent.com/8732757/134700463-de5e0365-a747-4fbc-90bc-fa009cafe5fc.png">
<img width="503" alt="Screen Shot 2021-09-24 at 8 22 53 AM" src="https://user-images.githubusercontent.com/8732757/134700465-36444a31-29b6-48e5-b929-d28a0975da6b.png">